SourceEditorCompose/README.md

```diff
@@ -0,0 +1,47 @@
+# SourceEditor
+
+This sample contains migrated Source Editor which is based on Jetpack Compose, designed for Surface Duo, Making use of the [dual view](https://docs.microsoft.com/dual-screen/introduction#dual-view) app pattern, users are able to edit and preview any changes simultaneously without switching windows.
+
+## Getting Started
+
+To learn how to load apps on the Surface Duo emulator, see the [documentation](https://docs.microsoft.com/dual-screen/android), and follow [the blog](https://devblogs.microsoft.com/surface-duo).
+
+## Features
+
+## Single Screen Mode
+
+In single screen mode, users are able to view one of two screens at a time: editor or preview. A button appears in the editor window that allows users to view a preview of the source code. 
+
+## Dual Screen Mode
+
+In dual screen mode, both windows are visible, so no buttons are visible to enable toggle.
+
+## Orientation
+
+This project supports various rotation configurations. In order to enable orientation changes within the application, ensure that rotation is enabled within the Duo emulator/device itself.
+
```

Comment:
Can you add screenshots of the app to each section here?

Reply:
uploaded!